src/objects/LOD.js

```diff
@@ -7,6 +7,7 @@ import { Object3D } from '../core/Object3D.js';
  * @author mrdoob / http://mrdoob.com/
  */
 
+var _currentLevel = 0;
```

Comment:
I don't think this code works when you define `_currentLevel` as a module scope variable. This will break as soon as you have more than on `LOD` object. `_currentLevel` has to be a real property of `LOD`.

Reply:
Oh, absolutely correct... however then I don't understand how you would get away without a getter in the current code ...

Reply:
I would just write `this. _currentLevel` in the constructor. It's more or less a convention that such properties are intended as private.

Reply:
Wilco